src/views/ErrorBoundary/PageNotFound.jsx

```diff
@@ -86,7 +86,7 @@ const PageNotFound = () => {
 					))}
 				</>
 				<div className="error-searchbar">
-					<form onSubmit={{executeSearch}}>
+					<form onSubmit={{executeSearch}} >
```

Comment:
The only change in this file is the extra space. Getting rid of the space should exclude this file from your PR.